sdk/formrecognizer/ai-form-recognizer/package.json

```
@@ -57,7 +57,7 @@
     "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1",
     "build:samples": "echo Obsolete.",
     "build:test": "tsc -p . && rollup -c 2>&1",
-    "build": "npm run clean && tsc -p . && rollup -c 2>&1 && api-extractor run --local",
+    "build": "tsc -p . && rollup -c 2>&1 && api-extractor run --local",
```

I believe we decided to get rid of the prebuild script and embed the clean one here:

```suggestion
    "build": "npm run clean && tsc -p . && rollup -c 2>&1 && api-extractor run --local",
```